src/core/flate_stream.js

```diff
@@ -148,6 +149,50 @@ class FlateStream extends DecodeStream {
     this.codeBuf = 0;
   }
 
+  get isAsync() {
+    return true;
+  }
+
+  async getImageData(length, _ignoreColorSpace) {
+    const data = await this.asyncGetBytes();
+    if (!data) {
+      return this.getBytes(length);
+    }
+    return data.subarray(0, length);
```

Comment:
Why not order as follows?
```suggestion
    if (data) {
      return data.subarray(0, length);
    }
    return this.getBytes(length);
```

Reply:
Or perhaps just as a one-liner instead?
```suggestion
    return data ? data.subarray(0, length) : this.getBytes(length);
```